common/translations/lang/en.json

```diff
@@ -786,6 +786,35 @@
     "DOWNLOAD_APP_DESCRIPTION": "Please download the MyCrypto Desktop app so you can securely complete creating your new account and start managing your funds.",
     "DOWNLOAD_APP_DOWNLOAD_BUTTON": "Download for",
     "DOWNLOAD_APP_FOOTER_INFO": "Not sure what this is?",
-    "DOWNLOAD_APP_FOOTER_INFO_LINK": "Learn more about our desktop app."
+    "DOWNLOAD_APP_FOOTER_INFO_LINK": "Learn more about our desktop app.",
+    "SCREEN_LOCK_NEW_HEADING":"Create Password to Lock Your Wallet",
+    "SCREEN_LOCK_NEW_DESCRIPTION":"We want to help you keep your funds safe! Please create a password so you can securly view your wallet when you come back to MyCrypto.",
+    "SCREEN_LOCK_NEW_PASSWORD_LABEL":"Password (min 8 characters)",
+    "SCREEN_LOCK_NEW_CONFIRM_PASSWORD_LABEL":"Confirm password",
+    "SCREEN_LOCK_NEW_CREATE_PASSWORD_BUTTON": "Create Password",
+    "SCREEN_LOCK_NEW_PASSWORD_TOO_SHORT":"Password must be at least 8 characters long",
+    "SCREEN_LOCK_NEW_PASSWORDS_DONT_MATCH":"Passwords don't match",
+    "SCREEN_LOCK_FORGOT_PASSWORD_HEADING":"Forgot Screen Lock Password?",
```

Comment:
I think this should just be "Forgot Password?"

Reply:
There is "Forgot Password" in designs but "Forgot Screen Lock Password" in Jira card. We can change it to whatever you think is better.

Reply:
I would prefer "Forgot Screen Lock Password" so users don't get confused with their keystore file password (or Facebook password 😉😅😂).